effekt/shared/src/main/scala/effekt/generator/chez/ChezScheme.scala

```diff
@@ -59,16 +59,16 @@ trait ChezScheme {
   /**
    * Entrypoint used by the LSP server to show the compiled output
    */
-  def compileSeparate(input: AllTransformed)(using C: Context) =
-    C.using(module = input.main.mod) { Some(chez.PrettyPrinter.format(compile(input.main))) }
```

Comment:
This module is also used for error reporting positions.

Reply:
Ah, TIL. Then we have to keep it, of course.

Reply:
But if we do not use error reporting anymore, it can of course be dropped.